### DISCUSSION TITLE
Workload Page - Recommendation Tile - Unable to populate Image

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#102] [Q&A] [answered]
Hi Team, 

Upon executing the latest GitHub code of the Microsoft-Fabric-workload-development-sample application, we noticed the recommendation tile images were not populating. Please do find the attached screenshot for reference.  Kindly let us know if there are any changes we need to make on our end.
<img width="959" alt="Workload_Image" src="https://github.com/user-attachments/assets/38059c87-b306-4f11-81d5-664c8caa1814">

Thank you

**Comment by ramizsha** [comment, marked as answer]
@SanchanaMohankumar we have fixed this issue, can you check again?

**Reply by SanchanaMohankumar** [reply]
Hi @ramizsha, we are able to see the images in recommendation tile now. Thank you 
<img width="928" alt="image" src="https://github.com/user-attachments/assets/3fda246b-ffdb-4895-9a6e-6d4cbd4c69fd">
